liquidator/liquidator.js

```diff
@@ -133,9 +133,9 @@ class Liquidator {
     // The `price` is a BN that is used to determine if a position is liquidatable. The higher the
     // `price` value, the more collateral that the position is required to have to be correctly collateralized.
     // Therefore, we add a buffer by deriving scaledPrice = price * (1 - crThreshold)
-    const scaledPrice = this.fromWei(
-      price.mul(this.toBN(this.toWei("1")).sub(this.toBN(this.toWei(this.crThreshold.toString()))))
-    );
+    const scaledPrice = price
+      .mul(this.toBN(this.toWei("1")).sub(this.toBN(this.toWei(this.crThreshold.toString()))))
+      .div(this.toBN(this.toWei("1")));
```

Comment:
@mrice32 @chrismaree I replaced the `.fromWei` with `.div(toBN(toWei(1))` at the end.

Reply:
nit: does it matter that this is now a `BN` rather than a string?

Reply:
No we always either `.toString()` it or `toBN()` it